setup.py

```diff
@@ -32,7 +32,7 @@
 ] + tests_requires
 
 install_aiohttp_requires = [
-    "aiohttp==3.7.1",
+    "aiohttp==3.7.2",
```

Comment:
I think the change should be `aiohttp>=3.7.0` as you mentioned this changed was fixed at `3.7.0`.

Reply:
> I think the change should be `aiohttp>=3.7.0` as you mentioned this changed was fixed at `3.7.0`.

The 3.7.0 version is buggy (See PR #156) so this is probably a bad idea.
Should we request a specific version of aiohttp or should we use `aiohttp>=3.7.1` ???

Reply:
Ah yes, there was a different bug regarding `typing-extensions` as well.

If that's the case, we must use a range for aiohttp like as `aiohttp>=3.7.1,<3.8.0` so any patch version can be used with this library and also avoid any breaking change from a minor release.

Reply:
@KingDarBoja I think all request changes was already applied, let me know if i miss something.